gtsam/geometry/Point3.h

```diff
@@ -68,8 +68,8 @@ GTSAM_EXPORT Point3 mean(const CONTAINER& points) {
   return sum / points.size();
 }
 
-/// mean of Point3 pair 
-GTSAM_EXPORT Point3Pair mean(const std::vector<Point3Pair>& abPointPairs);
+/// mean of Point3 pair
```

Comment:
`means` in the docstring, and better docstring maybe?

Reply:
Done